drivers/include/ds75lx.h

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (C) 2019 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_ds75lx Maxim DS75LX temperature sensor
+ * @ingroup     drivers_sensors
+ * @brief       Device driver interface for the Maxim DS75LX temperature sensor
+ *
+ * @{
+ *
+ * @file
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ */
+
+#ifndef DS75LX_H
+#define DS75LX_H
+
+#include "saul.h"
+#include "periph/i2c.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Thermometer resolution
+ */
+typedef enum {
+    DS75LX_RESOLUTION_9 = 0,        /**< 9 bits resolution, 25ms max conversion time */
```

Comment:
why don't you map this directly to `25`, `50` and so on milliseconds here, instead of going through this indirection of 0-3 here and then remap to the values in the driver?

Reply:
Because this enum also corresponds to the value stored in the configuration register.

Reply:
ah, okay

However, according to the [datasheet](https://datasheets.maximintegrated.com/en/ds/DS75LX.pdf) resolution times are 25, 50, 100, and **200** (not 250, for the latter). So it's doubling with every bit, and hence you don't need the array `_max_conversion_times` but could instead just to `25 << dev->params.resolution`